README.md

```diff
@@ -149,6 +155,36 @@ API URL of the GitHub server.
 
 The path for the sync configuration file.
 
+### `gpg_username`
+
+**Required:** `false`  
+**Default:** ``
+
+The username to use for the signature
+
+### `gpg_email`
+
+**Required:** `false`  
+**Default:** ``
+
+The email to use for the signature
+
+### `gpg_private_key`
+
+**Required:** `false`  
+**Default:** ``
+
+The private key created for the user with the username and email provided: `-----BEGIN PGP PRIVATE KEY BLOCK-----
+...
+-----END PGP PRIVATE KEY BLOCK-----`
+
+### `gpg_passphrase`
+
+**Required:** `false`  
+**Default:** ``
+
+The passphrase to read the key
+
```

Comment:
In the readme, can you add a dedicated section about commit signature ?

It does not have to be long, but just an example of the few commands/instrcutions (or links to official doc) needed to generate working paramete'rs

Reply:
Added a section at the end